packages/core/content-releases/admin/src/components/tests/CMReleasesContainer.test.tsx

```diff
@@ -155,6 +150,7 @@ describe('CMReleasesContainer', () => {
     render();
 
     await screen.findByRole('complementary', { name: 'Releases' });
+    await screen.findByRole('button', { name: /release action options/i });
```

Comment:
how come we need this?

Reply:
[ReleaseActionMenu](https://github.com/strapi/strapi/blob/301d2027a59183bfa76012748eaeef6b5ffae9d1/packages/core/content-releases/admin/src/components/ReleaseActionMenu.tsx#L224) calls useRBAC, which makes an API call. This call would be resolved after the test is done, causing the act error.

When the API call resolves, the "release action options" button appears, so I wait for that inside the test